### DISCUSSION TITLE
4-D and 5-D arrays in TIME.utils.peaks_to_RGB

### DISCUSSION THREAD

**Discussion by Hyedryn** [DelinteNicolas/UNRAVEL#8] [Ideas]
Hi,

Would it be possible to accept 5-D arrays instead of list of 4-D arrays and single 4-D arrays for standalone volume in TIME.utils.peaks_to_RGB() ? 

This would permit a better integration with other diffusion MRI oriented software such as DiPy or ElikoPy. 

Thanks in advance,

Quentin

**Comment by DelinteNicolas** [comment]
Hello,

A 'safety net' has been added to accept 4D arrays as input and 5D arrays in the shape of (x,y,z,3,k).

The changes will be uploaded with version 0.0.4.

Nicolas
